frontend/src/components/button/index.js

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled'
+import { fontSize, lineHeight, space, color, fontWeight } from 'styled-system'
+import tag from 'clean-tag'
+
+export const BaseButton = styled(tag.button)`
+  font-family: ${({ theme }) => theme.fontSans};
+  margin: 0;
+  border: 0;
+  &:focus {
```

Comment:
A thought: it might be an idea to pass a focus color as a prop so that the focus color can be customized if needed, with this color as the default prop?

Reply:
So I'm wondering if we want this vs having the focus color set in the theme and having all our components get it from there?

Reply:
Getting the focus colour from the theme seems to be more in line with the general model IMO.

Reply:
I was thinking of not adding that here, but rather in a separate PR (and there we'd also use it for other relevant components)